src/OrchardCore.Modules/Orchard.Tokens.Content/Services/ISlugService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orchard.Tokens.Content.Services
+{
+    public interface ISlugService
```

Comment:
Move to Orchard.Tokens.Content.Abstractions project. You'll need to create it.
If we use a service here, it's because we might want other modules to register alternative implementations. By providing an abstractions project these other modules won't have to reference the Module itself.